website/docs/index.md

```diff
@@ -28,8 +28,8 @@ We welcome contributions of all kinds! Whether you’re fixing bugs, adding new
 
 * Fork the repository.
 * Create a new branch for your changes.
-* Setup a [local development environment](/development/quickstart/).
-* Read the [developers guide](/development/).
+* Setup a [local development environment](development/quickstart.md).
+* Read the [developers guide](development/index.md).
```

Comment:
Are you sure that the website pages can be addressed with these URLs terminating in ".md"?

Reply:
I've checked it locally.

Reply:
Have you checked locally with "make generate-docs"?

Reply:
`cd website && mkdocs build && mkdocs serve`